tests/ast/test_natspec.py

```diff
@@ -7,6 +7,7 @@
 test_code = """
 '''
 @title A simulator for Bug Bunny, the most famous Rabbit
+@license MIT
```

Comment:
Should do a couple variations here (and => suggested renderings):
- Missing license field => `""`
- Empty license field `@license` => `""`
- `@license Apache-2.0` => `"Apache-2.0"`
- `@license Apache-2.0 OR MIT` => `"Apache-2.0 OR MIT"`
- `@license Apache-2.0 AND MIT` => `"Apache-2.0 AND MIT"`
- `@license Apache-2.0 AND (MIT OR GPL-2.0-only)` => `"Apache-2.0 AND (MIT OR GPL-2.0-only)"`

The latter 3 cases are there for the future if we decide to process these further

Reply:
Okay.